wallettool/src/test/java/org/bitcoinj/wallettool/WalletToolTest.java

```diff
@@ -0,0 +1,49 @@
+package org.bitcoinj.wallettool;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import picocli.CommandLine;
+
+import static org.junit.Assert.*;
+
+/**
+ * Basic functional/integration tests of {@code wallet-tool}
+ */
+public class WalletToolTest {
```

Comment:
Shouldn't this be called `CommandLineTest`?

Reply:
@schildbach Do you have a name suggestion or should we stick with `WalletToolTest`?

Reply:
> I don't think so because `CommandLine` is a class from Pico CLI.

You're right. I completely misread the code.